src/groups.py

```diff
@@ -1,11 +1,5 @@
 import pygame
-from src.settings import (
-    LAYERS,
-    SCREEN_HEIGHT,
-    SCREEN_WIDTH,
-    Coordinate,
-)
-
+from src.settings import *
```

Comment:
Wildcard imports will make the linter complain, so it's best that this gets reverted

Reply:
I will go back through and import only the required imports.